static/js/emoji_picker.js

```diff
@@ -19,6 +19,22 @@ var search_results = [];
 var section_head_offsets = [];
 
 function get_emoji_categories() {
```

Comment:
I was mainly doubtful regarding if that function actually returns all the categories or not.